plugins/Morpheus/javascripts/ajaxHelper.js

```diff
@@ -279,11 +279,24 @@ function ajaxHelper() {
         if(status == 'abort') {
             return;
         }
-        $('#loadingError').show();
-        setTimeout( function(){
-            $('#loadingError').fadeOut('slow');
-        }, 2000);
-    };
+
+        var loadingError = $('#loadingError');
+        if (Piwik_Popover.isOpen()) {
+            if (deferred && deferred.status === 500) {
+                $(document.body).html(deferred.responseText);
+            } else {
+                // Just display a generic error message
+                Piwik_Popover.setContent(loadingError.clone().html());
+            }
+        } else {
+            loadingError.show();
+            setTimeout( function(){
+                loadingError.fadeOut('slow');
+            }, 200000000);
```

Comment:
If the timeout is going to be this big, is it even necessary to have it?

Reply:
I reckon we could use same timeout as before?

Reply:
Have removed the timeout since it doesn't really make sense for it to fade out anyway.